rubocop-minitest.gemspec

```diff
@@ -25,5 +25,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "minitest", "~> 5.11"
```

Comment:
The latest version of minitest is specified.
https://rubygems.org/gems/minitest/versions
Is there any reason not to support older versions?

Reply:
No reason. I just picked it from the rubygems. Can you let me know which version we should go with? I can see Rails 4.0 has minitest `4.7.5`

Reply:
Thanks! Let's see how it goes :-)